.gitignore

```diff
@@ -46,19 +49,16 @@ build/
 .gradle
 local.properties
 *.iml
+.cxx/
+*.keystore
```

Comment:
Leave these in, we do ship a debug keystore and never have access to the prod one, only the CI has.